### ISSUE TITLE
Middleware breaks flask_wtf.csrf

### ISSUE THREAD

**Issue by P-T-I** [Danangjoyoo/flask-http-middleware#8]
Using the very basic of middlewares:
```
class MetricsMiddleware(BaseHTTPMiddleware):
    def __init__(self):
        super().__init__()

    def dispatch(self, request, call_next):
        response = call_next(request)
        return response
```
CSRF tokens do no longer match and effectively disables the proper functionality of the app. 
using:
flask==2.3.3
Werkzeug~=2.3.7
flask-login~=0.6.2
flask-wtf~=1.2.1
WTForms~=3.1.0

Any pointers?

**Comment by danjoyboytvlk** [comment]
Hi @P-T-I I think this one is similar with #4 can you please try it?

**Comment by P-T-I** [comment]
@danjoyboy Thanks for your reply; I checked that issue; but I believe mine is different; the CSRF protection if not something that is enforced by another type of middleware that is overwritten.

**Comment by Danangjoyoo** [comment]
@P-T-I Hi, sorry takes long regarding this, I've released the new version `0.4.3`, please kindly try it, thanks!
